ui-super-heroes/src/main/docker/Dockerfile.jvm

```diff
@@ -0,0 +1,93 @@
+@@ -0,0 +1,93 @@
+####
+# This Dockerfile is used in order to build a container that runs the Quarkus application in JVM mode
+#
+# Before building the container image run:
+#
+# ./mvnw package
+#
+# Then, build the image with:
+#
+# docker build -f src/main/docker/Dockerfile.jvm -t quarkus/quinoa-app-jvm .
+#
+# Then run the container using:
+#
+# docker run -i --rm -p 8080:8080 quarkus/quinoa-app-jvm
```

Comment:
`quarkus/quinoa-app-jvm` _should_ be `quarkus/ui-super-heroes-jvm17` (to be consistent with all the other apps)